cookiecutter/tap-template/{{cookiecutter.tap_id}}/.github/workflows/{% if cookiecutter.include_cicd_sample_template == 'GitHub' %}ci_workflow.yml{%endif%}

```diff
@@ -22,8 +22,6 @@ jobs:
         python-version: {{ '${{ matrix.python-version }}' }}
     - name: Install Poetry
       uses: snok/install-poetry@v1
-      with:
-        version: 1.1.8
```

Comment:
@visch Can a compatible version be used instead, i.e. something like `1.1`?

Future versions of Poetry could have breaking changes.

Reply:
Tried that here https://github.com/MeltanoLabs/target-postgres/pull/15/files , no good https://github.com/MeltanoLabs/target-postgres/actions/runs/2685763367 

Going to implement this slightly different, but same result as sem ver to 1.1 and simpler imo

Reply:
Done, what do you think about not using the action at all? Same number of lines, but we mess with the system site-packages